andela/vale-ah-backend
# #164139767 User Registration with JWT

#### What does this PR do?
Sends a JWT token on successful user registration
#### Description of Task to be completed?
- write unit test
- create route
- validate inputs with `joi`
- create controller
#### How should this be manually tested?
- ensure that the build is passing on travis 
- test the `POST /api/users` endpoint with Postman at the [review app](https://vale-ah-backend-staging-pr-12.herokuapp.com/)
#### Any background context you want to provide?
Users once registered via the registration endpoint, should receive a JWT to be required on all subsequent calls to all other endpoints that require authentication
#### What are the relevant pivotal tracker stories?
[#164139767](https://www.pivotaltracker.com/story/show/164139767)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
